src/components/ExpandedMobMenu.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
- [ ] Let's don't disable linter rules. These rules are here to help us write besr code.

Reply:
Please enlighten me on how to do this for React projects. If do you look at "all" projects module 3 React, to work everyone did this, otherwise doesn't work.